src/styles/utils/colors.scss

```
@@ -0,0 +1,11 @@
+html {
+  --main-color: #1d1d1d;
+  --secondary-color: #fff;
+  --menu-bg-color: #c2c2c2;
+  --button-hover-shadow-color: #e8e8f066;
+  --button-active-shadow-color: #e8e8f0a3;
+  --input-bg-color: #f8f8fa;
+  --input-accent-color: #292929;
+  --input-placeholder-color: #7e7e83;
+  --input-hover-border-color: #d5d7de;
+}
```

replace ur `CSS` variables with `SCSS` variables
```suggestion
$main-color: #1d1d1d;
...
```

---

So, my mentor mentioned that it's better to use CSS variables for colors, because then it will be easier to do dark/light theme. That's why I used these particular ones